app-android/build.gradle.kts

```diff
@@ -26,6 +26,13 @@ android {
         targetCompatibility = JavaVersion.VERSION_11
     }
 
+    packagingOptions {
+        exclude ("META-INF/DEPENDENCIES")
```

Comment:
I've always wondering what this stuff is for? Is this generated or you had to add manually to fix a build issue?

Reply:
Manually to fix a build issue. My naive understanding is that multiple libraries have the same metadata file name and clash